src/v2/Components/Footer/FooterDownloadAppBanner.tsx

```diff
@@ -0,0 +1,90 @@
+import React from "react"
+import {
+  Box,
+  Column,
+  Flex,
+  GridColumns,
+  Image,
+  Text,
+  TextVariant,
+  useThemeConfig,
+} from "@artsy/palette"
+import { resized } from "v2/Utils/resized"
+
+const BANNER_IMAGE_SRC = "https://files.artsy.net/consign/banner-large.jpg"
+const QR_CODE_SRC = "https://files.artsy.net/images/artsy-qr-code.svg"
+
+export const FooterDownloadAppBanner = () => {
+  const resizedImg = resized(BANNER_IMAGE_SRC, {
+    height: 320,
+    quality: 85,
```

Comment:
@icirellik I bumped the quality from `70` to `85`; its still sub-100k; the JPEG artifacting was very noticeable IMO at `70`